src/components/footer/EsFooter.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <footer :class="`${$router.currentRoute.fullPath.includes('pro') ? 'is-pro' : ''}`">
-    <div class="wave footer" />
+    <div :class="['wave', 'footer', isProWave ? 'is-pro-wave' : '', isClientWave ? 'is-client-wave' : '', isContactWave || isFAQWave ? 'is-common-wave' : '']"/>
```

Comment:
Can you create a computed property and put this logic there?

Something like:

getFooterClasses() {
return {
'jancsi': true,
'rocknroll': 1>2 ? true : false
'manele': sunt cocalar? true : false
};
},